packages/vsx-registry/src/browser/vsx-extensions-contribution.ts

```diff
@@ -22,8 +22,13 @@ import { Widget } from '@theia/core/lib/browser/widgets/widget';
 import { VSXExtensionsModel } from './vsx-extensions-model';
 import { ColorContribution } from '@theia/core/lib/browser/color-application-contribution';
 import { ColorRegistry, Color } from '@theia/core/lib/browser/color-registry';
-import { TabBarToolbarContribution, TabBarToolbarRegistry } from '@theia/core/lib/browser/shell/tab-bar-toolbar';
+import { TabBarToolbarContribution, TabBarToolbarItem, TabBarToolbarRegistry } from '@theia/core/lib/browser/shell/tab-bar-toolbar';
 import { FrontendApplicationContribution, FrontendApplication } from '@theia/core/lib/browser/frontend-application';
+import { MessageService, Mutable } from '@theia/core/lib/common';
+import { FileDialogService, OpenFileDialogProps } from '@theia/filesystem/lib/browser';
+import { WorkspaceService } from '@theia/workspace/lib/browser';
```

Comment:
I think we should avoid coupling `@theia/workspace` to `@theia/vsx-registry`, the `workspace` extension is also not listed as a dependency in the `package.json`.